spec/models/embeddable/labbook_answer_spec.rb

```diff
@@ -41,4 +41,19 @@
       expect(a_request(:post, labbook_replace_url)).to have_been_made.once
     end
   end
+
+  describe '#show_in_runtime?' do
+    describe 'with a disabled labbok' do
+      let(:labbook) { stub_model(Embeddable::Labbook, show_in_runtime?: false) }
+      it "should return false" do
+        expect(labbok_answer.show_in_runtime?).to eql(false)
```

Comment:
Since you are in this file could you fix the typo "labbok_answer" => "labbook_answer"?